packages/tailwindcss/src/utilities.ts

```diff
@@ -381,10 +367,10 @@ export function createUtilities(theme: Theme) {
       supportsFractions?: boolean
     } = {},
   ) {
-    utilities.static(`${name}-px`, (candidate) => {
-      if (!supportsNegative && candidate.negative) return
-      return handle(candidate.negative ? '-1px' : '1px')
-    })
+    if (supportsNegative) {
+      utilities.static(`-${name}-px`, () => handle('-1px'))
+    }
+    utilities.static(`${name}-px`, () => handle('1px'))
```

Comment:
Wondering if this should use `staticUtility` for consistency now, since we also use the `functionalUtility` helper for the case below (Is it even necessary to add the `staticUtility` abstraction now in the first place though? 🤔)